src/evict/evict_page.c

```diff
@@ -665,8 +671,14 @@ __evict_review(
 			 * suggests trying the lookaside table.
 			 */
 			if (F_ISSET(cache, WT_CACHE_EVICT_LOOKASIDE) &&
-			    !F_ISSET(conn, WT_CONN_EVICTION_NO_LOOKASIDE))
+			    !F_ISSET(conn, WT_CONN_EVICTION_NO_LOOKASIDE)) {
+				if (__wt_random(&session->rnd) % 10 == 1) {
```

Comment:
Should check `F_ISSET(cache, WT_CACHE_EVICT_DEBUG_MODE) && __wt_random(...`

Reply:
Good catch. Thanks.